packages/jerni/src/dev-cli/appendEventsToMarkdown.ts

```diff
@@ -0,0 +1,48 @@
+import fs from "node:fs/promises";
+import hash_sum from "hash-sum";
+import type { Code, Yaml } from "mdast";
+import { fromMarkdown } from "mdast-util-from-markdown";
+import { frontmatterFromMarkdown, frontmatterToMarkdown } from "mdast-util-frontmatter";
+import { toMarkdown } from "mdast-util-to-markdown";
+import { frontmatter } from "micromark-extension-frontmatter";
+import yaml from "yaml";
+import type { ToBeCommittedJourneyEvent } from "../types/events";
+import getEventsFromAst from "./getEventsFromAst";
+
+export default async function appendEventsToMarkdown(filePath: string, events: ToBeCommittedJourneyEvent[]) {
+  const fileContent = await fs.readFile(filePath, { encoding: "utf8" });
+
+  // parse the file content as markdown using import { fromMarkdown } from "mdast-util-from-markdown";
+  const ast = fromMarkdown(fileContent, {
+    extensions: [frontmatter(["yaml"])],
+    mdastExtensions: [frontmatterFromMarkdown(["yaml"])],
+  });
+
+  // append the events to the markdown, event node are jsonc with meta ?type=events
+  for (const event of events) {
+    const eventNode = {
+      type: "code",
+      lang: "jsonc",
+      value: JSON.stringify(event),
+      meta: "?type=events",
```

Comment:
in the future I think we can use event checksum as a meta tag. That way we can detect if an individual event has changed.

Reply:
`?` prefix feels extraneous, better to add it in the parsing logic, not to the readable file.